src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -96,7 +96,7 @@ export default class SpotTileModel extends ModelBase {
   @observeEvent('popOutTile')
   _onPopOutTile() {
     this._log.info(`Popping out tile`);
-    this._regionManagerHelper.popout(332, 190);
+    this._regionManagerHelper.popout(`${this.currencyPair.symbol} Spot`, 332, 155);
```

Comment:
There is a to do on that method to change this


Sent from Outlook Mobile






On Tue, Apr 26, 2016 at 6:34 AM -0700, "Dimitar Christoff" <notifications@github.com> wrote:





> @@ -96,7 +96,7 @@ export default class SpotTileModel extends ModelBase {
>    @observeEvent('popOutTile')
>    _onPopOutTile() {
>      this._log.info(`Popping out tile`);
> -    this._regionManagerHelper.popout(332, 190);
> +    this._regionManagerHelper.popout(`${this.currencyPair.symbol} Spot`, 332, 155);

these hard values will not scale well for responsive design - the view should pass these.

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/303/files/ebceeaf1d5e0ff6ffdb3c5c8fe31d926d5e31f4e#r61085523